hwdb.d/60-seat.hwdb

```diff
@@ -0,0 +1,20 @@
+# This file is part of systemd.
+#
+# This file lists graphic devices that don't have a DRM driver and fall back to
+# a frame-buffer one instead. Since commit 6260d28b8a, frame-buffer devices are
+# no more considered as graphical capable.
+#
+# The matches have the 'fb' prefix to make sure that only the framebuffer
+# device, and not the (parent) PCI device, is tagged with 'master-of-seat'.
+#
+# Allowed properties are:
+#    ID_TAG_MASTER_OF_SEAT=1
+
+#########################################
+# eXtreme Graphic Innovation (XGI)
+#########################################
+
+# Z7/Z9 (XG20 core), a rather ancient graphic chip, doesn't have a DRM driver
```

Comment:
maybe say that this is virtualized by some VM manager though. because as I understood this is the reason we are having the discussion

Reply:
AFAIK XGI Z7/Z9 is a real HW unlike the 2 other cases (HyperV and Parallels).

Reply:
@hramrach  said otherwise:

https://github.com/systemd/systemd/pull/20070#issuecomment-872055380

he said this was about virtualization. So what is it now?

Reply:
Nevermind, HyperV was only mentioned in the bugreport as an example of another thing that is also broken.